packages/vehicle-rental-overlay/src/VehicleRentalOverlay.story.js

```diff
@@ -0,0 +1,259 @@
+    const id = `Station ${idx}`;
+    return {
+      ...makeBaseRandomRentalStation({ id, networks }),
+      bikesAvailable: randomInt(20),
+      spacesAvailable: randomInt(20)
+    };
+  }
+
+  // make a free-floating style bike rental station
+  const id = `Bike ${idx}`;
+  return {
+    ...makeBaseRandomRentalStation({ id, networks }),
+    bikesAvailable: 1,
+    isFloatingBike: true,
+    spacesAvailable: 0
+  };
+}
+
+function makeRandomCarRentalStation(mapVal, idx) {
+  // make a free-floating style car rental station
+  const id = `Bike ${idx}`;
+  const networks = ["CAR2GO"];
+  return {
+    ...makeBaseRandomRentalStation({ id, networks }),
+    carsAvailable: 1,
+    isFloatingCar: true,
+    spacesAvailable: 0
+  };
+}
+
+function makeRandomEScooterRentalStation(mapVal, idx) {
+  // make a free-floating style car rental station
+  const id = `E-scooter ${idx}`;
+  const networks = ["SHARED"];
+  return {
+    ...makeBaseRandomRentalStation({ id, networks }),
+    isFloatingVehicle: true,
+    spacesAvailable: 0,
+    vehiclesAvailable: 1
+  };
+}
+
+const bikeMapSymbols = [
+  {
+    dockStrokeColor: "#000000",
+    fillColor: "#FF2E28",
+    maxZoom: 13,
+    minZoom: 0,
+    pixels: 4,
+    type: "circle"
+  },
+  {
+    dockStrokeColor: "#000000",
+    fillColor: "#FF2E28",
+    maxZoom: 17,
+    minZoom: 14,
+    pixels: 6,
+    type: "circle"
+  },
+  {
+    maxZoom: 999,
+    minZoom: 18,
+    type: "hubAndFloatingBike"
+  }
+];
+const carMapSymbols = [
+  {
+    fillColor: "#009cde",
+    maxZoom: 13,
+    minZoom: 0,
+    pixels: 4,
+    type: "circle"
+  },
+  {
+    fillColor: "#009cde",
+    maxZoom: 17,
+    minZoom: 14,
+    pixels: 6,
+    type: "circle"
+  },
+  {
+    fillColor: "#009cde",
+    maxZoom: 999,
+    minZoom: 18,
+    type: "marker"
+  }
+];
+const configCompanies = [
+  {
+    id: "BIKETOWN",
+    label: "Biketown",
+    modes: "BICYCLE_RENT"
+  },
+  {
+    id: "CAR2GO",
+    label: "car2go",
+    modes: "CAR_RENT"
+  },
+  {
+    id: "RAZOR",
+    label: "Razor",
+    modes: "MICROMOBILITY_RENT"
+  },
+  {
+    id: "SHARED",
+    label: "Shared",
+    modes: "MICROMOBILITY_RENT"
+  }
+];
+const EScooterMapSymbols = [
+  {
+    fillColor: "#F80600",
+    maxZoom: 13,
+    minZoom: 0,
+    pixels: 4,
+    strokeColor: "#CCCCCC",
+    type: "circle"
+  },
+  {
+    fillColor: "#F80600",
+    maxZoom: 17,
+    minZoom: 14,
+    pixels: 6,
+    strokeColor: "#CCCCCC",
+    type: "circle"
+  },
+  {
+    fillColor: "#F80600",
+    maxZoom: 999,
+    minZoom: 18,
+    type: "marker"
+  }
+];
+const randomBikeRentalStations = Array(20)
+  .fill(null)
+  .map(makeRandomBikeRentalStation);
+const randomCarRentalStations = Array(20)
+  .fill(null)
+  .map(makeRandomCarRentalStation);
+const randomEScooterStations = Array(20)
+  .fill(null)
+  .map(makeRandomEScooterRentalStation);
+const setLocation = action("setLocation");
+
+class ZoomControlledMapWithVehicleRentalOverlay extends Component {
```

Comment:
Could this be achieved with just a story showing a different initial zoom level for the layer?

Reply:
Alternatively, you could set up a knob for the storybook user to change the zoom interactively.